src/FakerDotNet/Fakers/PokemonFaker.cs

```diff
@@ -0,0 +1,39 @@
+using FakerDotNet.Data;
+
+namespace FakerDotNet.Fakers
+{
+    public interface IPokemonFaker
+    {
+        string Name();
+        string Location();
+        string Move();
+    }
+    
+    internal class PokemonFaker : IPokemonFaker
+    {
+        private static readonly PokemonData Data = new PokemonData();
+        
+        private readonly IFakerContainer _fakerContainer;
+
+        public PokemonFaker(IFakerContainer fakerContainer)
+        {
+            _fakerContainer = fakerContainer;
+        }
+
+
```

Comment:
Can you remove the extra carriage return? The rest of the classes don't have this format

Reply:
Done